frontend/package.json

```diff
@@ -16,8 +16,10 @@
   "dependencies": {
     "eslint-plugin-playwright": "^1.6.0",
     "pinia": "^2.1.7",
+    "v-tooltip": "^2.1.3",
```

Comment:
You don't need this dependency, neither vuedraggable anymore. To remove than, you should just get rid of these lines, and than run `yarn install`